packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -452,6 +452,17 @@ async function parseRequestObject(requestObject: IDataObject) {
 		axiosConfig.maxRedirects = 0;
 	}
 
+	if (axiosConfig.maxRedirects !== 0) {
+		axiosConfig.beforeRedirect = (redirectedRequest) => {
+			if (axiosConfig.headers?.Authorization) {
```

Comment:
do we really need the `if` checks? if axios is explicitly removing these (part of the reason why we have this PR), I'd assume the `if` checks are unnecessary.

Reply:
The inner if checks are necessary, if I remove them it throws because of headers being set to `undefined`. I guess the `axiosConfig.maxRedirects !== 0` check is not needed as beforeRedirect won't be called if there is no redirect. wdyt remove it?

Reply:
oh yeah. the `maxRedirects` check can go.